packages/did/src/DidLinks/AccountLinks.chain.ts

```diff
@@ -132,6 +135,21 @@ function isLinkableAccountId(
   return 'isAccountId32' in arg && 'isAccountId20' in arg
 }
 
+function accountFromChain(
+  account: AccountId32,
+  networkPrefix = ss58Format
+): KiltAddress | SubstrateAddress {
+  if (isLinkableAccountId(account)) {
```

Comment:
How can we call `isLinkableAccountId` on an argument which is already an `AccountId32`?

Reply:
I think it's probably the `Array<StorageKey<[AccountId32, AccountId32]>>` type down in the `connectedAccountsFromChain` function.